src/main/webapp/src/app/ranking/ranking-item/ranking-item.component.html

```diff
@@ -1,8 +1,18 @@
-<div>
-  <img height="200" src={{hiddenGem.photo}} alt={{hiddenGem.name}}>
-  <p id="name">{{hiddenGem.name}}</p>
-  <p id="business_type">{{hiddenGem.business_type}}</p>
-  <p id="address">{{hiddenGem.address}}</p>
-  <p id="price_level">{{hiddenGem.price_level}}</p>
-  <p id="rating">{{hiddenGem.rating}}</p>
-</div>
+<mat-card class="mat-elevation-z2">
+  <mat-card-header>
+    <mat-card-title id="name">{{hiddenGem.id}}. {{hiddenGem.name}}</mat-card-title>
+    <mat-card-subtitle id="address">{{hiddenGem.address}}</mat-card-subtitle>
+  </mat-card-header>
+  <mat-card-content>
+    <mat-grid-list cols="2" rowHeight="200px">
+      <mat-grid-tile>
+        <div id="business-info">
+          <p class="mat-subheading-2" id="business_type"><b>Business type:</b> {{hiddenGem.business_type}}</p>
+          <p class="mat-subheading-2" id="price_level"><b>Price level:</b> {{hiddenGem.price_level}}</p>
+          <p class="mat-subheading-2" id="rating"><b>Rating:</b> {{hiddenGem.rating}}</p>
```

Comment:
maybe you don't need the "mat-subheading-2" class  if you can used the id="business-info" for the div (the business info tag is used in .css but not mat-subheading-2)? unless you are calling this class somewhere else. I think the class and id are grouping the same information.

Reply:
Similarly to the mat-card class, this is a material stylesheet class (https://material.angular.io/guide/typography). Removing it will remove the typography styling on the card